am-lib/src/main/java/uk/gov/hmcts/reform/amlib/enums/SecurityClassification.java

```diff
@@ -19,4 +22,11 @@ public int getHierarchy() {
     public boolean isVisible(int maxHierarchy) {
         return maxHierarchy >= this.getHierarchy();
     }
+
+    public static SecurityClassification valueOf(int hierarchy) {
```

Comment:
Java docs. As we refactored and made it as public

Reply:
👍